src/diffusers/schedulers/scheduling_unclip.py

```diff
@@ -100,6 +100,15 @@ class UnCLIPScheduler(SchedulerMixin, ConfigMixin):
         prediction_type (`str`, default `epsilon`, optional):
             prediction type of the scheduler function, one of `epsilon` (predicting the noise of the diffusion process)
             or `sample` (directly predicting the noisy sample`)
+        thresholding (`bool`, default `False`):
```

Comment:
Can we run the UnCLIP slow tests here to make sure we didn't break anything? Also @williamberman could you give this a look here?